ui/src/components/Sessions/index.js

```diff
@@ -1,14 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
 import {Link} from "react-router-dom";
-
-import "./style.scss";
-
+import DeleteIcon from '@material-ui/icons/Delete'
 import {CSSTransition, TransitionGroup,} from 'react-transition-group';
+import { Observable } from "rxjs";
+import "./style.scss";
+import CircularProgress from '@material-ui/core/CircularProgress';
 
 const Sessions = (props) => {
     const {sessions = {}} = props;
     const list = Object.keys(sessions);
 
+    const [loading, setLoading] = useState([])
+
+    const deleteSession = (e, id) => {
+      e.preventDefault()
+      e.stopPropagation()
+      setLoading([...loading, id])
+
+      const deleteSession = Observable.ajax({
+        url: `/wd/hub/session/${id}`,
+        method: 'DELETE',
+      })
+
+      deleteSession.subscribe(
+        res => {
+          console.log(res)
```

Comment:
please don't log anything in console until its really the only way to give a feedback. Here is not that case